src/apiQueries/useCountries.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from "@tanstack/react-query";
+import { API_URL } from "constants/settings";
+import { fetchApi } from "helpers/fetchApi";
+import { ApiCountry, AppError } from "types";
+
+export const useCountries = () => {
+  const query = useQuery<ApiCountry[], AppError>({
+    queryKey: ["countries"],
+    queryFn: async () => {
+      return await fetchApi(`${API_URL}/countries`);
+    },
+    // Keeping the fetched data for longer since it won't change that often
+    staleTime: 5 * 60 * 100,
```

Comment:
just to confirm - is this  meant to be 30 seconds or 5 minutes? 

Might be a little clearer to write `30 * 1000` (or 5 * 60 * 1000, if it's supposed to be 5 minutes)

Reply:
That's a typo, I meant 5 minutes. Thanks for catching it! :pray: